docs/modules/kafka.md

```diff
@@ -82,3 +91,34 @@ The `Brokers(ctx)` method returns the Kafka brokers as a string slice, containin
 <!--codeinclude-->
 [Get Kafka brokers](../../modules/kafka/kafka_test.go) inside_block:getBrokers
 <!--/codeinclude-->
+
+#### BrokersByHostDockerInternal
+
+The `BrokersByHostDockerInternal(ctx)` method returns the Kafka brokers as a
+string slice, containing the hostname `host.docker.internal` and a random port
+defined by Kafka's public port (`19092/tcp`).
```

Comment:
suggestion: clarify what the 19092 port is, it's not clear on reading if its an example of random port or is that re-enforcing the standard port